antora.yml

```diff
@@ -8,20 +8,23 @@ asciidoc:
   attributes:
     product_name: OpenShift Serverless Logic
     context: OpenShift Serverless Logic
-    kogito_version_redhat: 1.24.0.Final-redhat-00011
+    kogito_version_redhat: 1.27.0.Final-redhat-00001
```

Comment:
can be updated to `1.27.0.Final-redhat-00005`

Reply:
Done!